zigbee/XBee_Sensor_Simulator.py

```diff
@@ -126,6 +138,16 @@ def enqueue(self, packet, to=None):
                     "to": to_id
                 })
 
+    def discover(self, callback):
+        """
+        Discover other XBee devices in the network.
+
+        This method is only used on the ground station in the control panel
+        to refresh the status of the other XBee devices.
+        """
+
+        pass
```

Comment:
I guess I cannot really test it right now with the XBee simulator ;) I'm not sure how difficult it is to implement the discovery for the simulators. Could you just send a special packet/command to all the socket ports (given we know the maximum number of sensors beforehand) and respond to that with another special command? Then it would be quite similar to the physical version.